swap_meet/item.py

```diff
@@ -0,0 +1,38 @@
+
+
+class Item:
+    def __init__(self, category = None):
+        
+        if category == None:
+            self.category = ""
+            
+        else:
+            self.category = category
```

Comment:
we can shorten this up by letting the default parameters do their job instead of the if-else statement.also, it doesn't look like you've created the `condition` category! you refer to it on line 21, so we need that to be part of the Item class.

```suggestion
    def __init__(self, category ="", condition=0):
            self.category = category
            self.condition=condition
```